test/framework/toolchain.py

```diff
@@ -1066,61 +1066,66 @@ def test_rpath_args_script(self):
         script = find_eb_script('rpath_args.py')
 
         # simplest possible compiler command
-        out, ec = run_cmd("%s gcc '' -c foo.c" % script, simple=False)
+        out, ec = run_cmd("%s gcc '' '%s/lib,%s/lib64,$ORIGIN' -c foo.c" % (script, self.test_prefix, self.test_prefix), simple=False)
```

Comment:
Construct the argument separately, since you're using it in multiple tests.
This also avoids making the line too long.

```python
rpath_inc = '%(prefix)s/lib,%(prefix)s/lib64,$ORIGIN' % {'prefix': self.test_prefix}
out, ec = run_cmd("%s gcc '' '%s' -c foo.c" % (script, rpath_inc), simple=False)
```

And then reuse `rpath_inc` below too.